moon/docker/rover/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN echo "source /srcp2-competitors/ros_workspace/install/setup.bash" >> /root/.
 
 ENTRYPOINT ["./osgar/moon/docker/rover/entrypoint.bash"]
 
-CMD ["./osgar/moon/docker/rover/run_solution.bash"]
+CMD ["/bin/bash"]
```

Comment:
I am not sure I understand this change?

Reply:
CMD means the program to start inside the docker by default (ie if docker is started without command line parameters indicating what inside the docker should be launched). The existing line made no sense because running the solution without indicating which round gives an error. And you apparently cannot supplement just part of the parameters from command line (e.g., have the script name in CMD but pass the round number as a parameter to docker on launch). So I decided to launch bash by default and the test script just then passes the full  `./osgar/moon/docker/rover/run_solution.bash -r $ROUND` to docker

Reply:
`CMD` is actually argument to `ENTRYPOINT` so sometimes that can be used to gain some advantage as well